draft-demarco-oauth-status-attestations.md

```diff
@@ -361,6 +366,32 @@ Below a non-normative example of an HTTP Response with an error.
   }
 ~~~
 
+Since the Status Assertion Request is an array of strings, if one or more status
+assertion request are made for a Digital Credential with errors,
+the response status code will be always `200` including the parameters as specified in  the OAuth 2.0 RFC
+[Section 5.2](https://tools.ietf.org/html/rfc6749#section-5.2).
+
+In addition to the error codes referenced in {{RFC6749}}, the following error codes
+MUST also be supported:
```

Comment:
no

RFC6749 works in a different way, we can only say that the status assertion error object data schema is inspired by the errors defined in RFC6749, using error and error_description within the payload